src/main/java/racingcar/domain/Cars.java

```diff
@@ -24,8 +25,33 @@ public void updateAll(RandomNumberGenerator randomNumberGenerator){
   public String currentCarsPosition() {
     StringBuilder sb = new StringBuilder();
     for(Car car : cars) {
-      sb.append(car.positionToLine()).append("\n");
+      sb.append(car.getName()).append(" : ").append(car.positionToLine()).append("\n");
     }
     return sb.toString();
   }
+
+  public List<String> getWinner() {
+    int max = 0;
+    List<String> winnerNames = new ArrayList<>();
+
+    getWinnerNames(winnerNames, getMaxPositionInCars(max));
+
+    return winnerNames;
+  }
+
+  private void getWinnerNames(List<String> winnerNames, int max) {
+    for(Car car : cars) {
+      if(car.getPosition() == max) {
+        winnerNames.add(car.getName());
+      }
+    }
+  }
+
+  private int getMaxPositionInCars(int max) {
+    for(Car car : cars) {
+      if(max < car.getPosition())
+        max = car.getPosition();
+    }
```

Comment:
아래와 같은 구현도 가능하긴 합니다(참고만 해 주세요).

```suggestion
    return cars.stream().mapToInt(Car::getPosition).max();

Reply:
아 말씀해주신 부분처럼 수정을 했는데 수정 후 push를 하니까 merge conflict가 또 다시 발생해서 미처 파악하지 못한 부분인거 같습니다. 앞으로 리베이스나 머지에 대해서 좀 더 신경을 써서 진행하도록 하겠습니다.